dev/cognitiveservices/data-plane/Language/analyzeconversations-authoring.json

```diff
@@ -3010,6 +3769,14 @@
     }
   },
   "parameters": {
+    "ConversationalAnalysisAuthoringArmTokenHeaderParameter": {
+      "in": "header",
+      "name": "armToken",
+      "description": "The user's ARM token.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
```

Comment:
The user token making the request just can't pass through? If not, other services for connected storage (BYOS) tend to pass a client ID/secret or even AAD token (which this likely is already). Or is this typical of Language? I haven't seen this pattern in Language yet.

Reply:
We removed the armToken now